test_dask_scheduler is broken

```
::test_ray_dask_basic PASSED                                             [ 50%]
::test_ray_dask_persist FAILED                                           [100%]
=================================== FAILURES ===================================
____________________________ test_ray_dask_persist _____________________________
ray_start_regular_shared = {'metrics_export_port': 63505, 'node_id': 'd0adb89d1727bd8d418d26784b39a70672c99abd085d665a5f61e308', 'node_ip_address': '10.30.0.177', 'object_store_address': '/tmp/ray/session_2021-02-05_22-04-52_308315_18423/sockets/plasma_store', ...}
    def test_ray_dask_persist(ray_start_regular_shared):
        arr = da.ones(5) + 2
        result = arr.persist(scheduler=ray_dask_get)
>       np.testing.assert_array_equal(result.dask.values()[0], np.ones(5) + 2)
E       TypeError: 'dict_values' object does not support indexing
/home/travis/build/ray-project/ray/python/ray/tests/test_dask_scheduler.py:38: TypeError
=========================== short test summary info ============================
FAILED ::test_ray_dask_persist - TypeError: 'dict_values' object does not sup...
========================= 1 failed, 1 passed in 6.43s ==========================
================================================================================
==================== Test output for //python/ray/tests:test_dask_scheduler:
============================= test session starts ==============================
platform linux -- Python 3.6.12, pytest-5.4.3, py-1.10.0, pluggy-0.13.1 -- /opt/miniconda/bin/python3
cachedir: .pytest_cache
rootdir: /home/travis/.cache/bazel/_bazel_travis/b88c129a127452fc94033a29d9f90e20/execroot/com_github_ray_project_ray/bazel-out/k8-opt/bin/python/ray/tests/test_dask_scheduler.runfiles/com_github_ray_project_ray
plugins: rerunfailures-9.1.1, timeout-1.4.2, asyncio-0.14.0, sugar-0.9.4
collecting ... collected 2 items
::test_ray_dask_basic PASSED                                             [ 50%]
::test_ray_dask_persist FAILED                                           [100%]
=================================== FAILURES ===================================
____________________________ test_ray_dask_persist _____________________________
ray_start_regular_shared = {'metrics_export_port': 62394, 'node_id': '78ff0762c68255a09c776e9d2849849b8f04cfff8c196b77d9eaa79f', 'node_ip_address': '10.30.0.177', 'object_store_address': '/tmp/ray/session_2021-02-05_22-04-59_804715_18632/sockets/plasma_store', ...}
    def test_ray_dask_persist(ray_start_regular_shared):
        arr = da.ones(5) + 2
        result = arr.persist(scheduler=ray_dask_get)
>       np.testing.assert_array_equal(result.dask.values()[0], np.ones(5) + 2)
E       TypeError: 'dict_values' object does not support indexing
```

https://travis-ci.com/github/ray-project/ray/jobs/480870501

--- a/python/ray/tests/test_dask_scheduler.py
+++ b/python/ray/tests/test_dask_scheduler.py
@@ -36,7 +36,8 @@ def test_ray_dask_persist(ray_start_regular_shared):
     arr = da.ones(5) + 2
     result = arr.persist(scheduler=ray_dask_get)
     np.testing.assert_array_equal(
-        next(iter(result.dask.values())), np.ones(5) + 2)
+        next(iter(result.dask.values())),
+        np.ones(5) + 2)
 
 
 if __name__ == "__main__":

--- a/python/ray/tests/test_dask_scheduler.py
+++ b/python/ray/tests/test_dask_scheduler.py
@@ -35,7 +35,8 @@ def test_ray_dask_basic(ray_start_regular_shared):
 def test_ray_dask_persist(ray_start_regular_shared):
     arr = da.ones(5) + 2
     result = arr.persist(scheduler=ray_dask_get)
-    np.testing.assert_array_equal(next(iter(result.dask.values())), np.ones(5) + 2)
+    np.testing.assert_array_equal(
+        next(iter(result.dask.values())), np.ones(5) + 2)
 
 
 if __name__ == "__main__":

--- a/python/ray/tests/test_dask_scheduler.py
+++ b/python/ray/tests/test_dask_scheduler.py
@@ -35,7 +35,7 @@ def test_ray_dask_basic(ray_start_regular_shared):
 def test_ray_dask_persist(ray_start_regular_shared):
     arr = da.ones(5) + 2
     result = arr.persist(scheduler=ray_dask_get)
-    np.testing.assert_array_equal(result.dask.values()[0], np.ones(5) + 2)
+    np.testing.assert_array_equal(next(iter(result.dask.values())), np.ones(5) + 2)
 
 
 if __name__ == "__main__":

